File: go/wam/5460.go
const WamMessagingFavoritesUpdate = 5460 // channel: regular

type MessagingFavoritesUpdateEvent struct {
	ContactFavCountAfterUpdate int64 `wam:"contact_fav_count_after_update"` // field 1
	ContactFavCountBeforeUpdate int64 `wam:"contact_fav_count_before_update"` // field 2
	FavoritesUpdateEntryPoint FAVORITESUPDATEENTRYPOINT `wam:"favorites_update_entry_point"` // field 3
	GroupFavCountAfterUpdate int64 `wam:"group_fav_count_after_update"` // field 4
	GroupFavCountBeforeUpdate int64 `wam:"group_fav_count_before_update"` // field 5
}

File: ts/wam/5460.ts
export const WamMessagingFavoritesUpdate = 5460 as const

export interface MessagingFavoritesUpdateEvent {
  /** field 1, wire `contact_fav_count_after_update` */
  contactFavCountAfterUpdate?: number
  /** field 2, wire `contact_fav_count_before_update` */
  contactFavCountBeforeUpdate?: number
  /** field 3, wire `favorites_update_entry_point` */
  favoritesUpdateEntryPoint?: FAVORITESUPDATEENTRYPOINT
  /** field 4, wire `group_fav_count_after_update` */
  groupFavCountAfterUpdate?: number
  /** field 5, wire `group_fav_count_before_update` */
  groupFavCountBeforeUpdate?: number
}